contracts/BridgeLimiter.sol

```diff
@@ -119,6 +119,8 @@ contract BridgeLimiter is IBridgeLimiter, CommitteeUpgradeable, OwnableUpgradeab
 
         // update hourly transfers
         hourlyTransferAmount[_currentHour] += usdAmount;
+
+        emit HourlyTransferAmountUpdated(_currentHour, usdAmount);
```

Comment:
can we also emit current hour total value?

Reply:
yes

Reply:
we can but I will say this will add more computation on all transfer transactions. I feel like this calculation can happen on the indexing side offchain?

Reply:
fair